.travis.yml

```diff
@@ -1,8 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-    - 4.2
-    - 5
     - 6
```

Comment:
I think we should have testing on v8 and v10 right now.

Reply:
seems a little out-of-scope for this PR. maybe a separate PR for this?